app/javascript/dashboard/components/layout/Sidebar.vue

```diff
@@ -253,8 +263,24 @@ export default {
     this.$store.dispatch('notifications/unReadCount');
     this.$store.dispatch('teams/get');
     this.setChatwootUser();
+    document.addEventListener('keydown', this.handleKeyEvents);
```

Comment:
Use `eventListene` mixin after merging the PR https://github.com/chatwoot/chatwoot/pull/2672